prow/plugins.yaml

```diff
@@ -10,7 +10,9 @@ triggers:
   - istio/test-infra
   - istio/proxy
   - istio-releases/pipeline
+  - istio-ecosystem/authservice
   trusted_org: istio
+  trusted_org: istio-ecosystem
```

Comment:
I don t think that s valid YAML:
add another section:
```YAML
- repos
  - istio-ecosystem/authservice
  trusted_org: istio-ecosystem
```

Reply:
fixed